player.go

```diff
@@ -457,15 +438,16 @@ func (c *Client) PreviousOpt(opt *PlayOptions) error {
 // will cause the player to start playing the next song.
 //
 // Requires the ScopeUserModifyPlaybackState in order to modify the player state
-func (c *Client) Seek(position int) error {
-	return c.SeekOpt(position, nil)
+func (c *Client) Seek(ctx context.Context, position int) error {
+	return c.SeekOpt(ctx, position, nil)
 }
 
 // SeekOpt is like Seek but with more options
 //
 // Only expects PlayOptions.DeviceID, all other options will be ignored
-func (c *Client) SeekOpt(position int, opt *PlayOptions) error {
+func (c *Client) SeekOpt(ctx context.Context, position int, opt *PlayOptions) error {
```

Comment:
Should we add functional options here too?

Reply:
Good shout. I'll have a play with this now.

Reply:
I'm not sure the same benefits apply here as they do with instantiating the client, so I'll move ahead without functional options on SeekOpt/PlayOpt for now